### DISCUSSION TITLE
GitHub Actions latest is not synced with release latest

### DISCUSSION THREAD

**Discussion by keizer619** [community/community#49070] [Actions]
### Select Topic Area

Product Feedback

### Body

When we publish GitHub Action into market place for a patch version of previous minor, it will treat it as the latest version. When releasing such version we usually untick `Set as the latest release`

Here in GitHub latest version is [2201.4.0](https://github.com/ballerina-platform/ballerina-action/releases/tag/2201.4.0). However in the GitHub Actions latest version is[ 2201.3.3](https://github.com/marketplace/actions/ballerina-action)

**Comment by jsoref** [comment]
Please file a ticket to https://support.github.com/
